browser/src/UI/components/CursorPositioner.tsx

```diff
@@ -148,10 +148,15 @@ export class CursorPositionerView extends React.PureComponent<ICursorPositionerV
             visibility: this.props.hideArrow ? "hidden" : "visible",
         }
 
+        const leftSidePositioning = (x: number) =>  Math.sign(x) !== -1
```

Comment:
Looks like this could be simplified with `Math.abs`

Reply:
Thanks for the tip @bryphe, tweaked it to use `Math.abs`, much simpler!